pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -435,6 +438,9 @@ spec:
         {{$key}}: {{$value}}
 {{- end}}
 {{- end}}
+{{- if .cloudstackEtcdAffinity }}
+      affinity: '{{ .cloudstackEtcdAffinity }}'
```

Comment:
right now I see two very similar-sounding template substitution names: affinity and cloudstackAffinity. Could you make it more clear which one is for worker and which is for control plane?

Reply:
Done